www/src/components/Login.js

```diff
@@ -2,11 +2,12 @@ import { Box, Text } from 'grommet'
 import { Checkmark, StatusCritical } from 'grommet-icons'
 
 export function disableState(password, confirm) {
-  if (password.length === 0) return { disabled: true, reason: 'enter a password' }
-  if (password.length < 10) return { disabled: true, reason: 'password is too short' }
-  if (password !== confirm) return { disabled: true, reason: 'passwords do not match' }
+  if (password.length === 0) return { disabled: true, reason: '' }
```

Comment:
is there a reason some of the reasons were dropped?  I suppose this one is a bit obvious